specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Trigger.json

```diff
@@ -275,6 +275,66 @@
         "typeProperties"
       ]
     },
+    "BlobEventsTrigger":
+    {
+      "description": "Trigger that runs everytime a Blob event occurs.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MultiplePipelineTrigger"
+        }
+      ],
+      "properties": {
+        "typeProperties":
+        {
+          "description": "Blob Events Trigger properties.",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "blobPath": {
+              "description": "Path to container, folder, blob, or file extension for which events can trigger the pipeline.",

```

Comment:
Is this relative path? Can you add an example, or be more specific in the description?